bin/render-activity-stream-html.js

```diff
@@ -54,6 +54,22 @@ function getTextDirection(locale) {
   return RTL_LIST.includes(locale.split("-")[0]) ? "rtl" : "ltr";
 }
 
+function getScripts(options) {
+  const debugString = options.debug ? "-dev" : "";
+  return [
+    "chrome://browser/content/contentSearchUI.js",
+    "chrome://browser/content/contentTheme.js",
+    `${options.baseUrl}vendor/react${debugString}.js`,
+    `${options.baseUrl}vendor/react-dom${debugString}.js`,
+    `${options.baseUrl}vendor/prop-types.js`,
+    `${options.baseUrl}vendor/react-intl.js`,
+    `${options.baseUrl}vendor/redux.js`,
+    `${options.baseUrl}vendor/react-redux.js`,
+    `${options.baseUrl}prerendered/${options.locale}/activity-stream-strings.js`,
```

Comment:
oh wait. If we're loading the locale strings here, this can't be rendered "static" as it changes per locale.